mods/seasonal/statuses.js

```diff
@@ -171,6 +171,19 @@ let BattleStatuses = {
 			this.add(`c|@moo|/me moo`);
 		},
 	},
+	omroom: {
+		noCopy: true,
+		onStart: function (target, source) {
+			this.add(`c|%OM Room|use shift gear`);
+			this.add('-start', source, 'typeadd', 'Fairy');
+		},
+		onSwitchOut: function () {
+			this.add(`c|%OM Room|Ok brb I'm gonna (ladder Mix and Mega, roll battle some surv regs real quick)`);
```

Comment:
The phrases in ()s might be different options for him to say, I asked him for confirmation.

Reply:
Confirmed with OM Room